handlers/add_todo.go

```diff
@@ -0,0 +1,26 @@
+package handlers
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/knipferrc/plate-api/models"
+	"github.com/labstack/echo"
+)
+
+func (h *Handler) AddTodo(c echo.Context) error {
```

Comment:
I made this a pointer receiver vs value so it’s not making a copy of handler, I think it’s more efficient in this case.